### ISSUE TITLE
Confirming MIT license of code

### ISSUE THREAD

**Issue by EdwardDeaver** [nicmosc/brick-builder#5]
Hi, 

I'd like to build on top of this project. Just wanted to confirm the MIT license found in the package json is how this project is licensed? 

Best,
Edward

**Comment by corrigancd** [comment]
I tend to agree, but this is a great question, and a great project!! @nicmosc, do you have any thoughts on this one? :+1:

**Comment by nicmosc** [comment]
Hey @EdwardDeaver @corrigancd, indeed. I didn't put the licence text as at the time I didn't really care (oh younger me). I can include it in the project now if you wish to contribute/fork or anything of the sort. I don't have time right now to work on this, especially as it was mostly meant as a POC and would likely rewrite it from scratch today if I could 😅 

Let me know, and glad you like the project!

**Comment by corrigancd** [comment]
Hey @nicmosc, sure, it would be great if the repo could be licensed with MIT! I'm in the space of scoping for a fun new (to me) project and this might just be that! 

Thanks,
